doc/api/cli.md

```diff
@@ -932,6 +932,7 @@ V8 options that are allowed are:
 - `--perf-basic-prof-only-functions`
 - `--perf-prof`
 - `--perf-prof-unwinding-info`
+- `--interpreted-frames-native-stack`
```

Comment:
The flags are currently in alphabetical order. Can you move this one to line 930 to maintain alphabetical listing? Or is it better that it be below `--perf-*` flags?